lib/services/runtime/handlers/command.ts

```diff
@@ -34,7 +42,7 @@ export const getCommand = (runtime: Runtime, extractFrame: typeof extractFrameCo
     }
   }
 
-  const res = extractFrame<BaseNode.Command.Command>(runtime.stack, matcher);
+  const res = extractFrame<BaseNode.AnyCommonCommand>(runtime.stack, matcher);
```

Comment:
this was actually just cheating, because there are two types of commands: `Command` and `Intent`, we just pretended everything was `Command` because `Command` also has the `next` property.

But we need to verify that `res.command` is an `Intent.Command` to be able to access the `command.diagramID` property.

Alot of older programs/projects do not have the `Command.type` field so that is not reliable. We are forced to compare the properties.